rules/sort-switch-case.ts

```diff
@@ -229,11 +229,9 @@ export default createEslintRule<Options, MessageId>({
           if (b.some(node => node.isDefaultClause)) {
             return -1
           }
-          return compare({
-            a: a.at(0)!,
-            b: b.at(0)!,
-            options,
-          })
+
+          let defaultComparator = defaultComparatorByOptionsComputer(options)
```

Comment:
Let's move defaultComparatorByOptionsComputer to the top so that we don't have to calculate the comparator at each iteration of the comparison.